src/state/migrations/1.test.ts

```diff
@@ -0,0 +1,50 @@
+import { RouterPreference } from 'state/routing/types'
+import { UserState } from 'state/user/reducer'
+import { SlippageTolerance } from 'state/user/types'
+
+import { migration1, PersistAppStateV1 } from './1'
+
+const previousState: PersistAppStateV1 = {
+  user: {
+    userLocale: null,
+    userRouterPreference: RouterPreference.API,
+    userHideClosedPositions: false,
+    userSlippageTolerance: SlippageTolerance.Auto,
+    userSlippageToleranceHasBeenMigratedToAuto: true,
+    userDeadline: 1800,
+    tokens: {},
+    pairs: {},
+    timestamp: Date.now(),
+    hideBaseWalletBanner: false,
+  },
+  _persist: {
+    version: 0,
+    rehydrated: true,
+  },
+}
+
+describe('migration to v1', () => {
+  it('should migrate the default deadline', () => {
+    const result = migration1(previousState)
+    expect(result?.user?.userDeadline).toEqual(600)
```

Comment:
nit: can you also verify no other states have been changed?

Reply:
yes